coldfront/plugins/qumulo/views/update_allocation_view.py

```diff
@@ -90,32 +147,36 @@ def form_valid(
             end_date_extension=10,
         )
 
-        # NOTE - "storage_protocols" will have special handling
-        attributes_to_check = [
-            "cost_center",
-            "department_number",
-            "technical_contact",
-            "billing_contact",
-            "service_rate",
-            "storage_ticket",
-            "storage_quota",
-        ]
+        storage_quota_attribute = AllocationAttribute.objects.get(
+            allocation_attribute_type=AllocationAttributeType.objects.get(
+                name="storage_quota"
+            ),
+            allocation=allocation,
+        )
+
+        if storage_quota_attribute.value != form_data.get("storage_quota"):
+            AllocationAttributeChangeRequest.objects.create(
+                allocation_attribute=storage_quota_attribute,
+                allocation_change_request=allocation_change_request,
+                new_value=form_data.get("storage_quota"),
+            )
 
-        form_values = [form_data.get(field_name) for field_name in attributes_to_check]
+        storage_protocols = json.dumps(form_data.get("protocols"))
 
-        # handle "storage_protocols" separately
-        attributes_to_check.append("storage_protocols")
-        form_values.append(json.dumps(form_data.get("protocols")))
+        storage_protocols_attribute = AllocationAttribute.objects.get(
+            allocation_attribute_type=AllocationAttributeType.objects.get(
+                name="storage_protocols"
+            ),
+            allocation=allocation,
+        )
 
-        for attribute_name, form_value in zip(attributes_to_check, form_values):
-            UpdateAllocationView._handle_attribute_change(
-                allocation=allocation,
+        if storage_protocols_attribute.value != storage_protocols:
```

Comment:
This has wiped out the old attribute handling again.

In QA, I just generated an erroneous `storage_quota` change request again.

Reply:
https://coldfront-qa.ris.wustl.edu/allocation/change-request/27/

Reply:
This should be re-restored with the most recent updates.